django/db/migrations/operations/models.py

```diff
@@ -43,8 +43,11 @@ class CreateModel(ModelOperation):
 
     serialization_expand_args = ["fields", "options", "managers"]
 
-    def __init__(self, name, fields, options=None, bases=None, managers=None):
+    def __init__(
+        self, name, fields, options=None, bases=None, managers=None, state_only_op=None
+    ):
```

Comment:
The main objective of using `state_only_op` is to remove the need for new migration operations plus make the squashmigration much more easier . This [comment](https://github.com/django/django/pull/16585#issuecomment-1463964356) by @shaib explain the things better.

Reply:
I'm not suggesting to move all steps to a single operation, but only to add `MoveModelToApp` instead of using `CreateModel(..., state_only_op=True)`. Maybe the name `CreateModelInAppState` would be clearer.